### DISCUSSION TITLE
Will interaction with group items be supported in 5.0?

### DISCUSSION THREAD

**Discussion by crustycrabthe3rd** [Palakis/obs-websocket#802] [Q&A] [answered]
I find it frustrating that i can't do anything with scene items that are inside a group in 4.9.1. Will group interactions be supported in the new version?

**Comment by tt2468** [comment]
We are hoping to include intentional support for groups in some form or another. It is difficult though since groups are very broken internally in OBS, so having requests that can modify them has many risks.

**Comment by tt2468** [comment, marked as answer]
Update: Almost all Scene Item requests have group support in 5.x.

Notable exclusions are `CreateSceneItem` and `RemoveSceneItem`

**Reply by crustycrabthe3rd** [reply]
You the man, thank you! One more thing, with the current api, i can't change duplicate sources, can only change the base source, will that also be a limitation in 5.x?

**Reply by tt2468** [reply]
In v5, we switch to using the unique `sceneItemId` associated with items, so assuming you know the correct item's ID, you can interact specifically with it.

You can find the IDs of scene items in a scene by calling `GetSceneItemList`
